tests/InferTypesFuzz.elm

```diff
@@ -406,6 +406,10 @@ shrinkExpr expr =
             -- TODO take a stab at this? Do we actually even generate these?
             nope
 
+        CanonicalU.Shader _ ->
+            -- TODO take a stab at this? Do we actually even generate these?
```

Comment:
I wonder if we would have to make well-defined GLSL programs here. Let's leave it as it is right now :+1: